service/src/main/kotlin/xyz/lebkuchenfm/Application.kt

```diff
@@ -32,10 +38,22 @@ fun Application.module() {
     val songsRepository = SongsMongoRepository(database)
     val songsService = SongsService(songsRepository)
 
+    val eventStream = DummyEventStream() // TODO: To be replaced with actual WebSocket implementation.
+
+    val commandPrompt = environment.config.property("commandPrompt").getString()
```

Comment:
Maybe for now we should pass it like everywhere else and then make global refactor of config injecting.

Reply:
Configuration is a convention problem that we will have to solve for sure 👍